packages/sdk/src/solana/classes/helpers/nft-helper.ts

```diff
@@ -267,20 +263,21 @@ export class NFTHelper {
       throw new Error("No mint found for NFT");
     }
     const owner = await this.ownerOf(mint.address.toBase58());
-    return this.toNFTMetadataResolved(mint, owner, fullModel);
+    const supply = await this.supplyOf(mint.address.toBase58());
```

Comment:
i think we can promise.all this to speed things up a bit. Only thing to watch out for is RPC rate limiting when calling getAll() on a large-ish collection. gotta test that